features/pages/advanced_search_page.py

```diff
@@ -23,12 +23,12 @@ def __init__(self, driver):
         "inputTextWindow": ("XPATH", "//details/form/div/input[@class='form-control']"),
         "selectKeyList2": ("XPATH", "(//select[@class='form-select search-facet-field'])[2]"),
         "selectValueList2": ("XPATH", "(//select[@class='form-select'])[2]"),
-        "cve_idText": ("XPATH", "//tr[1]/td[2]/a"),
-        "impactText": ("XPATH", "//tr[1]/td[3]"),
-        "sourceText": ("XPATH", "//tr[1]/td[4]"),
-        "titleText": ("XPATH", "//tr[1]/td[6]"),
-        "workflow_stateText": ("XPATH", "//tr[1]/td[7]"),
-        "ownerText":  ("XPATH", "//tr[1]/td[8]"),
+        "cve_idText": ("XPATH", "//tr[1]/td[1]/a"),
+        "impactText": ("XPATH", "//tr[1]/td[2]"),
+        # "sourceText": ("XPATH", "//tr[1]/td[3]"),  # TODO remove unused
```

Comment:
Removing the source locator is ok since the source field has been removed from the flawlist.